transaction-manifest/src/lexer.rs

```diff
@@ -398,8 +398,8 @@ impl Lexer {
             "TAKE_NON_FUNGIBLES_FROM_WORKTOP" => Ok(TokenKind::TakeNonFungiblesFromWorktop),
             "RETURN_TO_WORKTOP" => Ok(TokenKind::ReturnToWorktop),
             "ASSERT_WORKTOP_CONTAINS" => Ok(TokenKind::AssertWorktopContains),
-            "TAKE_FROM_AUTH_WORKTOP" => Ok(TokenKind::TakeFromAuthWorktop),
-            "PUT_ON_AUTH_WORKTOP" => Ok(TokenKind::PutOnAuthWorktop),
+            "POP_FROM_AUTH_WORKTOP" => Ok(TokenKind::PopFromAuthWorktop),
+            "PUSH_ONTO_AUTH_WORKTOP" => Ok(TokenKind::PushOntoAuthWorktop),
```

Comment:
Perhaps better name would be:
`PUSH_AUTH`
`POP_AUTH`

Reply:
Haha, prefer the current ones. I'm sure we will get another chance of reviewing the entire instruction set.